src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,82 @@
+// Colors
+$grays: (
+  100: #f9fbfd,
```

Comment:
@carryall What if we use `hsl` instead of `rgb`?
I think it isn't just only about performance, it also easy to order the color 🤔

Reply:
@Lahphim Do you mean I should convert all of these values to `hsl` but wouldn't it easier to align with the `rgb` value from Figma or if I add the comment behind the value it's okay?

```scss
$grays: (
  100: hsl(210, 50%, 98%),  //#f9fbfd
  200: hsl(206, 26%, 95%),  //#eef2f5
  300: hsl(214, 16%, 92%),  //#e6e9ed
  400: hsl(224, 10%, 79%),  //#c3c6ce
  500: hsl(227, 8%, 65%),   //#9ea1ac
  600: hsl(229, 6%, 49%),   //#747784
  700: hsl(227, 8%, 33%),   //#4d505b
  800: hsl(230, 9%, 25%),   //#3b3d47
  900: hsl(234, 11%, 18%),  //#292a33
  backgound: hsl(240, 11%, 9%) //#15151a
);```

Reply:
@carryall yes like this 👍
It's a little bit hard but it looks easy to order amount of the color.
For example, we have have 2 rgb colors and we cannot make a decision that which one is darker 🤣

Reply:
✅  updated in [16cb735](https://github.com/carryall/survey-challenge/pull/2/commits/16cb735c3e01d450a83b200ace3880f02e2f1634)